openwisp_utils/api/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
```

Comment:
following the text of the other comment, here you could do something like:

```python
try:
    from rest_framework import serializers
except ImportError:  # pragma: nocover
    import sys
    print('Django REST Framework is required to use this feature but it is not installed')
    sys.exit(1)
```

Reply:
How about using django's abstraction for this?

```
from django.core.exceptions import ImproperlyConfigured

try:
    from rest_framework import serializers
except ImportError:  # pragma: nocover
    raise ImproperlyConfigured('Django REST Framework is required to use this feature but it is not installed')
```

Reply:
sure, a lot better! :+1: